frontend/src/upload.js

```diff
@@ -62,25 +89,33 @@ $(document).ready(function() {
       $('#link').attr('value', url);
       link.innerHTML = url;
       localStorage.setItem(info.fileId, info.deleteToken);
-      let del = document.createElement('td');
-      let btn = document.createElement('button');
-      btn.innerHTML = 'x';
-      btn.classList.add('delete-btn');
-      btn.addEventListener('click', e => {
+
+      // delete file
+      console.log($popupText.first());
+      $popupText.find('.del-file').click(e => {
         FileSender.delete(
           info.fileId,
           localStorage.getItem(info.fileId)
         ).then(() => {
-          e.target.parentNode.parentNode.remove();
+          e.target.parentNode.parentNode.parentNode.parentNode.remove();
           localStorage.removeItem(info.fileId);
         });
       });
-      del.appendChild(btn);
-      row.appendChild(del);
+
+      // show popup
+      btn.addEventListener('click', e => {
+        $popupText.toggleClass('show');
+      });
+      // hide popup
+      $popupText.find('.nvm').click(e => {
+        $popupText.toggleClass('show');
+      });
```

Comment:
these two have the same fn after the click event, perhaps we name the fn and reuse it?